chore(changelogs): add 1.7.0 changelogs (#164)
c6o/docs: content/changelogs/cli.md

This document contains the release notes for the CodeZero CLI.

## Release Notes for v1.7.0

> This release introduced a new daemon feature for querying a cluster's status and resources.  This is to support the upcoming `surf` interface (more information will be coming soon).

Reviewer: ```suggestion
This release introduced a new daemon feature for querying a cluster's status and resources.  This is to support the upcoming `surf` interface which is now live at [https://k8s.surf](https://k8s.surf).
```
